reactioncommerce/reaction
# Make order details panel visible.

Resolves #2617.

This PR resolves a bug with the orders page. Using the `Collapsed` mode to view the list of orders, the details panel for any specific order wasn't showing up because of some CSS issues.

## Test instructions
- Log in as admin and make one or two orders.
- Use the admin menu (on the right) to go to the `Orders` page.
- Switch to `Collapsed` view.
- Click the `>` beside any listed order and you'll notice that the details panel for that order now show up correctly in the browser.
- Try to resize the browser window. You'll notice that the order details panel stays visible throughout.
